README.md

```diff
@@ -7,7 +7,7 @@
 The structure of the whole Django project is built for you. Run the following commands in order to have your local environment up and running.  
 
 ```bash
-$ mkvirtualenv -p $(which python3) django_orm_intro
+$ mkvirtualenv -p $(which python3.5) django_orm_intro
```

Comment:
I think Cloud9 doesn't have Python3.5 installed. If we leave `python3` any version would work.

Reply:
C9 does have python 3.5, I've run this command. And if you install Django 2.1.2 on top of 3.4 (the default python3) it raises a ton of errors. I won't paste in the entire traceback because it's really long, but here's the last bit of it and what happens when you try to runserver:
![django_error_1](https://user-images.githubusercontent.com/36095717/48640314-f0906e00-e99b-11e8-92f9-23763739ded4.PNG)
![django_error_2](https://user-images.githubusercontent.com/36095717/48640320-f38b5e80-e99b-11e8-8c09-007065317046.PNG)

Reply:
Awesome, disregard my comments then.